src/Eccube/Resource/template/admin/Product/index.twig

```diff
@@ -110,6 +110,39 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
                 form.attr('action', $(this).data('action')).submit();
                 return false;
             });
+
+            var dataClass = [];
+            var modalClass = $('#product-classes');
+            $('form#form_bulk').on('click', 'table.table button.standardConfirm', function () {
+                var btnClass = $(this);
+                btnClass.attr('disabled', true);
+                var productId = btnClass.data('product-id');
+
+                if (dataClass[productId] != undefined) {
+                    renderClass(dataClass[productId], btnClass);
+                    return;
+                }
+
+                $.ajax({
+                    url: '{{ url('admin_product_classes_load') }}',
+                    type: 'POST',
+                    data: {'productId': productId},
+                }).done(function (data) {
+                    dataClass[productId] = data;
+                    renderClass(dataClass[productId], btnClass);
+                }).fail(function () {
+                    console.log('Failed');
```

Comment:
Please use `alert('Faild')` for now.
We have not decided how notify the Ajax error yet.